client/layout/logged-out.jsx

```diff
@@ -133,7 +133,7 @@ const LayoutLoggedOut = ( {
 				</div>
 			</div>
 			{ config.isEnabled( 'gdpr-banner' ) && <GdprBanner showGdprBanner={ showGdprBanner } /> }
-			{ sectionName === 'plugins' && (
+			{ ( sectionName === 'themes' || sectionName === 'theme' || sectionName === 'plugins' ) && (
```

Comment:
If we're dealing with more than 1 section, it might be more convenient to use [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes):

```suggestion
			{ [ 'themes', 'theme', 'plugins' ].includes( sectionName ) && (
```